config.yml

```diff
@@ -27,3 +27,4 @@ rancher-endpoint: ~
 
 node-scripts-location: 'scripts/vagrant'
 
+win-image-url: 'http://10.84.144.252/meera/ws2016.qcow2'
```

Comment:
can we set the default value of win-image-url to None and in the test check if it's not configured as None, then run the test?

Reply:
We'll need a conditional market for this. I'll implement it in a follow-on patch.

Reply:
ok sounds good.